src/jace/api.py

```diff
@@ -21,57 +23,80 @@
     from collections.abc import Callable, Mapping
 
 
-__all__ = ["grad", "jacfwd", "jacrev", "jit"]
+__all__ = ["JitOptions", "grad", "jacfwd", "jacrev", "jit"]
+
+# Used for type annotation, see the notes in `jace.stages` for more.
+_P = ParamSpec("_P")
+_RetrunType = TypeVar("_RetrunType")
```

Comment:
Thanks for spotting.

Actually, I like `_ReturnType` because it clearly expresses what I it is and in my view `_P` should be actually named differently. I named it that way because it was from the example.
However, I will think about it.

Reply:
Please try to follow Python idioms when writing Python code:  `_ReturnType` is a normal TypeVar without constraints or bounds and thus there is no reason to not name it with a short uppercase name like `_T` or `_R`, which makes very easy to see in any type specification that this is just a TypeVar.

Unconstrained TypeVars don't have any meaning, they are just placeholders for catching actual types in some expressions and they are only relevant in the applied function/class. Using the same TypeVar in different classes or functions does not involve any connection between the classes since the TypeVar represents different concrete types in different applied contexts.

A larger and more explicit name like `ReturnType` generally means that this is some kind of specific type like an actual concrete type, or a Protocol or a type alias to some union of types.